ecosystem-ci.ts

```diff
@@ -23,6 +24,14 @@ cli
 	.option('--commit <commit>', 'vite commit sha to use')
 	.option('--release <version>', 'vite release to use from npm registry')
 	.action(async (suites, options: CommandOptions) => {
+		if (options.commit) {
+			const url = `https://pkg.pr.new/${options.repo}/vite@${options.commit}`
+			const { status } = await fetch(url)
```

Comment:
should be able to use head request here to avoid download

Reply:
https://github.com/vitejs/vite/pull/18525#discussion_r1828964024